docs/docs/04-standard-library/01-cloud/queue.md

```diff
@@ -172,12 +172,12 @@ Purge all of the messages in the queue.
 ##### `push` <a name="push" id="@winglang/sdk.cloud.IQueueClient.push"></a>
 
 ```wing
-inflight push(message: str): void
+inflight push(messages: str): void
```

Comment:
Looks like a bug in how we're generating documentation in the `jsii-docgen` tool. I think we need some extra logic to make sure variadic parameters like this one are displayed as `...message: Array<str>`. Can we open an issue about this?

Reply:
Done: #3781